### PR TITLE
browser: v1.10.8, fixes to url matchers and dep upgrade

### PR DESCRIPTION
**Pull request by shazow**
Our JS regexps were using a single `\` escape instead of `\\`.

Also updated our deps. Will release an internal build for testing to make sure everything is still working.

### CODE REVIEW

**Comment by maxtaco** [comment]
Thanks for the quick turnaround!

**Comment by shazow** [comment]
Pushed v1.10.8 which throws an error when there is a duplicate querystring id (particularly for HN).

If anyone wants to test it before we release, you can sideload the extension from here: https://keybase.pub/shazow/keybase-extension/

**Comment by shazow** [comment]
@maxtaco Please re-review your approval.

File: browser/js/identities.js [hunk 1]
```diff
@@ -1,20 +1,25 @@
 // All of our identity services and matchers are defined here.
 
+// parseLocationQuery converts URL-encoded parameters into an object. It
+// requires unique keys, will throw an error if there is a duplicate key.
 function parseLocationQuery(s) {
```
Comment (shazow) [inline browser/js/identities.js]: Reformatted here, [use ?w=1 for a cleaner diff](https://github.com/keybase/client/pull/10277/commits/3edc650ea3a9fa789d239d84ea2cab3ea7780e54?w=1).

**Comment by shazow** [comment]
FYI this breaks Firefox. I suspect part of this bug was there for a reason, because Firefox uses a single-`/` escape for some of these regexps. @joshblum and I are looking into it.

**Comment by joshblum** [comment]
@shazow and I believe the bug is related to the latest FF version but that these changes don't make it worse.

**Pull request merged**